ci/docker/ray.cpu.base.wanda.yaml

```diff
@@ -1,8 +1,8 @@
 name: "ray-py$PYTHON_VERSION-cpu-base"
-froms: ["ubuntu:focal"]
+froms: ["ubuntu:22.04"]
```

Comment:
@aslonnie look like we need this as well to upgrade ray image to ubuntu:22.04

Reply:
no, this is what I said before, the wheel compiled on 22.04 does not work with conda because the libcxx version is too high.

Reply:
https://github.com/ray-project/ray/pull/45427

the conda runtime env related tests will fail

Reply:
this image is not for CI, this is the ray docker image we're releasing to customers

Reply:
the right way would be either use manylinux container to compile ray, or port manylinux compatible toolchain onto ubuntu 22.04 in some way.

Reply:
wait i'm a bit confused; is the rayproject/ray:2.xx image is already on ubuntu 22.04 or not yet?